### PR TITLE
Integrated Mouse Interactions (Domino Click | Table Click)

### PR DESCRIPTION
**Pull request by Gaby-Concepcion**
Finalized pick up and put down mouse interaction functions for the GameStatus and integrated them into the experimental game test.

### CODE REVIEW

**Review by AngelCIICMorales — CHANGES_REQUESTED** [review]
Hay conflicts

**Comment by Gaby-Concepcion** [comment]
Trate de guardar los cambios de los conflictos pero intento hacer merge. Voy a ver como puedo hacer el ajuste

**Review by AngelCIICMorales — APPROVED** [review]
LGTM

**Review by Gaby-Concepcion — COMMENTED** [review]
Changed conflict in GameStatus caused by a change in spacing

**Review by fernandopizarro1 — APPROVED** [review]
LGTM

File: frontend/src/gamestate/experimentalGame.js [hunk 1]
```diff
@@ -290,4 +290,22 @@ function MainGame() {
 
 }
 
+function onTableClick(tableElement, gameStatus) {
+    tableElement.addEventListener('click', (event) => {
+        const position = getPositionFromEvent(event);
```
Comment (sandy-marrero) [inline frontend/src/gamestate/experimentalGame.js:295]: This line is causing errors in the main branch. 

> ERROR in [eslint]
> src\gamestate\experimentalGame.js
> Line 295:26:  'getPositionFromEvent' is not defined  no-undef

**Comment by sandy-marrero** [comment]
This line is causing errors in the main branch. 

> ERROR in [eslint]
> src\gamestate\experimentalGame.js
> Line 295:26:  'getPositionFromEvent' is not defined  no-undef

**Comment by Gaby-Concepcion** [comment]
I'm taking a look right now!

**Comment by Gaby-Concepcion** [comment]
I took a look and honestly the easiest fix is a rollback. I could do a patch in the form of another merge request but I'm almost certain it will cause another conflict that would need to be reviewed and could still lead to an error somewhere. 

I would just say roll it back if possible, (if not I'll patch it and just merge that through), and we can just reopen the issue and leave it for next milestone or just delete it for it to be re-written so someone else who could perhaps do a better implementation can do so for the next milestone. As far as the points go, if it doesn't work on main then I don't mind just taking the L

**Comment by Gaby-Concepcion** [comment]
There's a patch out on "To Review" just in case there's an issue with rolling back. But if it's not needed it's fine to delete

**Pull request merged**